### DISCUSSION TITLE
2021-11-03 Weekly Meeting 28 Agenda / Notes

### DISCUSSION THREAD

**Discussion by mrmetaverse** [omigroup/omigroup#117] [Weekly Meetings]
Weekly Meeting 28 is happening on Wednesday November 3, 2021 from 9:00 AM PDT - 10:00 AM PDT

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR Discord](https://discord.gg/NJtT9grz5E).

## Agenda
   - Welcome new members and attendance
   - https://github.com/omigroup/omigroup/issues/72
   - glTF subgroup
   - WebXR Hackathon
   - https://github.com/omigroup/omigroup/issues/109
   - website
   - VKet Booth

   
 ## Add Ons
- Call for participation: Metaversity
- Getting in front of the noise, together

**Comment by mrmetaverse** [comment]
Welcome Adam! One of the founders of Open Sim. With Sinwave entertainment now, and has been with a number of metaverse companies/groups. 

Msub2 (Daniel) is with us today! Started the webxr discord. Passionate about webxr, and saw the webxr 

AeonixAeon (Will) is with us today! <3 fun fact: they named OMI!

**Reply by madjin** [reply]
Adam OG, 25 years in the space, was part of VRBA also: https://www.vrblockchainalliance.org/

Papers from AeonixAeon
https://drive.google.com/file/d/0B3u--BesWlB9Y1NhZDVMYXlFa2M/view
https://drive.google.com/file/d/0B3u--BesWlB9TFZEU3VSaHFxZzA/view?resourcekey=0-Ct-JzP6cJP9dY0G5iBgMTA
https://drive.google.com/file/d/0B3u--BesWlB9NUNHN2VqbVRrejg/view?resourcekey=0-_pXoznjgevCVU8yfbuuEuw

**Reply by AdamFrisby** [reply]
(Haha, thank you - yes. Been around the block a few times, I was in the older 2007 IETF VWRAP effort as well. These days I do sine.space.)

**Comment by mrmetaverse** [comment]
Evo mentioned that we are not a standards making body, although many of our contributors are from standards making bodies. We are a community serving as connecting tissue, testing ideas, pushing back, and implementing. We strive for neutrality and collectively creating protocols.

**Comment by mrmetaverse** [comment]
for gltf subgroup updates: [please see this meeting note from 10/28](https://github.com/omigroup/gltf-extensions/discussions/30)

**Comment by mrmetaverse** [comment]
## We need an implementation of OMI_audio_emitter on a different platform for it be able to pass the tests, that's the blocker

**Comment by mrmetaverse** [comment]
We voted and adopted our first draft Vision statement: _We bolster the metaverse as an open and interoperable resource for anyone, inspired by the collaborative efforts of the community._

![image](https://user-images.githubusercontent.com/63426722/140104558-a923eb6a-fc71-4fb5-a645-704820325412.png)
(there were 8 Y, 0N, and 9 Abstentions)

**Comment by mrmetaverse** [comment]
## WebXR Hackathon: 

:notebook: Notes are being taken [on notion](https://www.notion.so/WebXR-Hackathon-2ef41b9d287b4e7bb5ad007cde465bbd)

:calendar: No date set yet 

![image](https://user-images.githubusercontent.com/63426722/140104836-288aa702-2f2c-445a-9668-abad036d0b7b.png)

![image](https://user-images.githubusercontent.com/63426722/140105042-18c312bb-cacc-4fba-823b-159eb72bd762.png)

Would like to issue challenges related to key topics. I.e. accessibility : How might we make the metaverse more accessible?

**Reply by mrmetaverse** [reply]
![image](https://user-images.githubusercontent.com/63426722/140105768-fad86898-dde1-47ee-b0db-055ad4905497.png)

**Reply by mrmetaverse** [reply]
![image](https://user-images.githubusercontent.com/63426722/140106010-566ed7d9-484a-4499-a39f-936eb1c7d679.png)

**Reply by mrmetaverse** [reply]
Daniel is seeking help with a name. Toying with immersive web fest. 
Jin also suggested reaching out to Thrill and calling it something related to Nexus.

**Comment by mrmetaverse** [comment]
## Attendance 

![image](https://user-images.githubusercontent.com/63426722/140104955-0c1c3bf5-d5a6-462d-9b1e-1aadee6b8920.png)

**Comment by mrmetaverse** [comment]
## Metaversity CfP

Discussed [the Metaversity Call for Participation](https://github.com/omigroup/omigroup/issues/115). 

The goal is to flatten this out early, and get more contributors from a variety of backgrounds and orgs. We don't want this to turn into a lead/talent funnel for just one org. It's about helping others achieve thier goals of becoming a new or better metaverse creator. 

We believe this effort aligns with two of the OMI core values. 
![image](https://user-images.githubusercontent.com/63426722/140134587-65c51ffb-a898-46b7-8cb6-2b66b25d9dbb.png)

To get involved, please join us on the Metaversity Github Org, or reach out to @mrmetaverse directly with questions.

**Comment by mrmetaverse** [comment]
## Website group 

- Shadow is iterating on the style guide 

### We need help: Looking for an accessibility audit of our styleguide: https://github.com/omigroup/website/issues/39

**Comment by mrmetaverse** [comment]
Jin suggested a github repo as a shared asset library

**Comment by mrmetaverse** [comment]
Evo Proposed another demo night soon!

**Comment by mrmetaverse** [comment]
Moved https://github.com/omigroup/omigroup/issues/109 to next week
